app/views/restaurants/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_for @restaurant, url: {action: "create"} do |f| %>
+  <div>
+    Name: <%= f.text_field :name %>
+  </div>
+  <div>
+    Walking distance: <%= f.number_field :walking_minutes_away %>
+  </div>
+  <div>
+    Phone: <%= f.text_field :phone_number %>
+  </div>
+  <div>
+    Address: <%= f.text_field :street_address %>
+  </div>
+  <div>
+    Menu: <%= f.text_field :menu_link %>
```

Comment:
Ooohhhh! It's cause we aren't using SimpleForms! Nvmd lol I'm cool with not using simple forms 💯